i2c/ssd1306_i2c/ssd1306_i2c.c

```diff
@@ -387,13 +387,23 @@ int main() {
         "by the name of",
         "    PICO"
     };
-
```

Comment:
Minor nitpick, but could you add in this blank line again please?

Reply:
Done